src/state/slices/opportunitiesSlice/selectors/combined.ts

```diff
@@ -30,6 +33,37 @@ type GetOpportunityAccessorArgs = { provider: DefiProvider; type: DefiType }
 type GetOpportunityAccessorReturn = 'underlyingAssetId' | 'underlyingAssetIds'
 type GetOpportunityAccessor = (args: GetOpportunityAccessorArgs) => GetOpportunityAccessorReturn
 
+const makeClaimableStakingRewardsAmountFiat = ({
```

Comment:
Abstracted away for re-usability and to return 0 for LP earn opportunities, so we don't have to check for the opportunity type, giving us access to the rewards in the outer scope